madmom/features/tempo.py

```diff
@@ -898,11 +904,19 @@ def online_interval_histogram(self, activation, reset=True):
         # reset to initial state
         if reset:
             self.reset()
+        # select relevant activations from buffer for sutraction
+        buffer = self._buffer.buffer[self.min_interval:self.max_interval + 1]
```

Comment:
I'd prefer the variable called activations, because this is basically what it is.